app/controllers/concerns/arclight/ead_format_helpers.rb

```diff
@@ -0,0 +1,228 @@
+# frozen_string_literal: true
+
+module Arclight
+  ##
+  # A module to add EAD to HTML transformation rules for Arclight
+  module EadFormatHelpers # rubocop:disable Metrics/ModuleLength
+    extend ActiveSupport::Concern
+    include ActionView::Helpers::OutputSafetyHelper
+
+    def render_html_tags(args)
+      values = args[:value] || []
+      values.map! do |value|
+        html_sanitize(transform_ead_to_html(value))
+      end
+      values.map! { |value| wrap_in_paragraph(value) } if values.count > 1
+      safe_join(values.map(&:html_safe))
+    end
+
+    private
+
+    def transform_ead_to_html(value)
+      Loofah.xml_fragment(condense_whitespace(value))
```

Comment:
It looks like `.scrub!` is chainable; is there a reason we'd want to get the transformed-but-not-stripped HTML output? Or, could the `ead_to_html_scrubber` method take care of stripped all other elements?

Reply:
Fair question, but I think @seanaery made some good progress here and I would be ok kicking this to a future refactor.

Reply:
@cbeer Here's the refactor: PR #952 ... Doesn't seem to break anything :-)